mesonbuild/compilers/fortran.py

```diff
@@ -464,6 +464,27 @@ def __init__(self, exelist: T.List[str], version: str, for_machine: MachineChoic
     def language_stdlib_only_link_flags(self) -> T.List[str]:
         return ['-lflang', '-lpgmath']
 
+class LFortranFortranCompiler(FortranCompiler):
```

Comment:
`gfortran` -> `GnuFortranCompiler`
`lfortran` -> `LFortranFortranCompiler` ?

I believe a better class name for this (so that people don't wonder what a FortranFortran compiler is) would be `LLVMFortranCompiler`

Reply:
I'd like that but it isn't unambiguous. Flang is also an LLVM compiler (and part of LLVM core nowadays)

Reply:
Which we already support, and I'm pretty sure it's called either FlangFortran or ClangFortran

Reply:
Well, that seems annoying. :/ Dunno what to suggest for a better name.

Reply:
How about just `LFortranCompiler`?

Reply:
I think `LFortranCompiler` sounds good; thanks ^_^